junit5-jupiter-starter-bazel/junit5.bzl

```diff
@@ -21,36 +23,56 @@ JUNIT_EXTRA_DEPENDENCIES = [
     ("org.opentest4j", "opentest4j", "1.1.1"),
 ]
 
+JUNIT_SHA256_CHECKSUMS = {
+    "junit-jupiter-api": "3f476de9b214f20ca69da51e801186d001f67328a686df81bc3de3ba11953870",
+    "junit-jupiter-engine": "0eb1ab3fc8e4130943b54f4d86824b106ef1cd90d96789343f3944e48b3c501c",
+    "junit-jupiter-params": "52f7aeb346cfa41bb33a8d3dbab8c577f9c37f8f5a79a632af10b5c8f1e92186",
+    "junit-platform-commons": "341621f4d998fd7b539b38baa7e1a3da80b7cac00b983e6206b01c9290915fe9",
+    "junit-platform-console": "2a2407737b147004642ee8e65618d51cc3895fe03b9c42f54b07d480ce6b7190",
+    "junit-platform-engine": "23b41ac95e4673f7b31e8502424451be4154fe4db1d448448945e2215473c246",
+    "junit-platform-launcher": "d1ebfafa2bd87b05c7dce7249e1437a1c0e4f16af99d81f89c5a0c0d66dc1510",
+    "junit-platform-suite-api": "130bf4d60958e4345583e5804f4a2932c6b60832bf2bc797b9521ffe6c9acadc"
+}
+
```

Comment:
Unless there's some automation, I'd rather not add these here because it creates manual work whenever we release.

Reply:
Sounds good!

Reply:
We just have this house, feel like it is handy to take (maybe easier for the others too). Whatever works though.